search-app/src/cmr/search/api/request_context_user_augmenter.clj

```diff
@@ -38,12 +41,11 @@
   (let [{:keys [token]} context]
     (let [sids (cache/get-value (cache/context->cache context token-sid-cache-name)
                                 token
-                                #(or (sids-retriever/get-sids context token)
-                                     {:sids (acl/context->sids context)}))
-          token-guid (:guid sids)
-          sids (:sids sids)]
-      (when token-guid (info (format "Client token GUID: [%s]" token-guid)))
-      sids)))
+                                #(access-control/get-current-sids context token))
+          ;; If we are directly dealing with the results of an access-control query, parse the json
```

Comment:
What we store in the cache should be exactly the same as the return of the function. Why is it different?